### DISCUSSION TITLE
Unable to run the script file

### DISCUSSION THREAD

**Discussion by xxtemps** [ansys/pyansys-geometry#1058] [Q&A] [answered]
![1710471721835](https://github.com/ansys/pyansys-geometry/assets/12379914/cdec26b7-06a7-4df2-9a2c-3aa9c75d1457)
![1710471725856](https://github.com/ansys/pyansys-geometry/assets/12379914/3252034a-1ffc-408f-8f7b-2007adb0b415)
I'm using version 2024R1. I don't know what's wrong with it

**Comment by RobPasMue** [comment, marked as answer]
Hi @xxtemps - it seems to me that your script is wrong in some way... would it be possible to share the content of the script here? The error is not coming from PyAnsys Geometry itself, but from SpaceClaim.

**Reply by xxtemps** [reply]
Thank you. SpaceClaim's problem
